api/NuageAPI_test.go

```diff
@@ -959,3 +959,91 @@ func TestVMPowerOff(t *testing.T) {
 
 	vrsConnection.Disconnect()
 }
+
+//TestSplitActivation tests the split activation mode using SDK
+func TestSplitActivation(t *testing.T) {
+	vrsConnection, err1 := NewUnixSocketConnection(UnixSocketFile)
+	if err1 != nil {
+		t.Fatal("Unable to connect to the VRS")
+	}
+	enterpriseFetchingInfo := &bambou.FetchingInfo{Filter: "name == \"" + Enterprise + "\""}
```

Comment:
We have a util package that does fetching enterprises task for you; I would use that. Similar for domains and subnets.